jwst/engdblog/tests/test_engdblog.py

```diff
@@ -32,7 +32,7 @@ def test_barestring(caplog, engdb):
     assert mnemonic in result
     assert 'EngDBLogStep instance created' in caplog.text
     assert mnemonic in caplog.text
-    assert "Step EngDBLogStep running with args ('INRSI_GWA_Y_TILT_AVGED',).".format(mnemonic) in caplog.text
+    assert "Step EngDBLogStep running with args ('INRSI_GWA_Y_TILT_AVGED',{}).".format(mnemonic) in caplog.text
```

Comment:
Please remove this change.  This has been fixed elsewhere and is not related to this PR.